### ISSUE TITLE
Icalendar support (ICS)

### ISSUE THREAD

**Issue by niccokunzmann** [github/maintainermonth#88]
Hi,

I wanted to integrate this into my Android phone calendar and I was missing ICS (did not see it).
It would be nice to have a static ICS link to subscribe to.

Personally, I also maintain https://open-web-calendar.hosted.quelltext.eu
So, I was wondering whether you would like to embed a calendar that shows the events in the time zone of the viewer.

I don't know - these are my thoughts. What do you think?

Basically, I subscribe to this on Github to not forget about it but then, I do not like to read all the pull requests. (my personal problem)

**Comment by prathamshroff** [comment]
Sounds like a feature I'd like to have too, viewing all events in my Calender via a Subscribed iCal would be great!

**Comment by abbycabs** [comment]
Thansk @niccokunzmann & @prathamshroff -- feel free to send a PR!

**Comment by niccokunzmann** [comment]
The framework is react. One might define this next to index.js.
There is already api/events.js and index.js iterates over the events.
The only addition needed is something that creates ICS.

**Comment by abbycabs** [comment]
Hey @niccokunzmann! I appreciate your research! I also didn't build the initial site, so your investigations are helpful :)

**Comment by ashleywolf** [comment]
Addressed in #380 (merged Mar 23). `api/calendar.js` adds ICS generation using the `ics` library, and the schedule page now has a download button for the .ics file. Thanks @niccokunzmann for the original request!
